templates/signedKeyMail.tmpl

```diff
@@ -0,0 +1,10 @@
+Hi!
+
+We successfully verified your GPG key.
+Attached to this email you will find your public key signed by the validation server.
+
+If you did not request this, please ignore this email.
```

Comment:
At this point we should probably not say "Ignore this email", since someone has probably gained access to that account.